include/rdma/fabric.h

```diff
@@ -336,6 +336,13 @@ enum {
 	FI_PROTO_CXI_RNR,
 };
 
+enum {
+	FI_TAG_BITS,
+	FI_TAG_HPC,
+	FI_TAG_AI,
```

Comment:
Are we wanting HPC and AI or MPI and CCL?  The latter are closer aligned to what the definitions correspond to.

Reply:
MPI and CCL. Forgot to change that.